go/vt/callinfo/plugin_grpc.go

```diff
@@ -64,6 +64,17 @@ func (gci *gRPCCallInfoImpl) Text() string {
 	return fmt.Sprintf("%s:%s(gRPC)", gci.remoteAddr, gci.method)
 }
 
-func (gci *gRPCCallInfoImpl) HTML() template.HTML {
-	return template.HTML("<b>Method:</b> " + gci.method + " <b>Remote Addr:</b> " + gci.remoteAddr)
+func (gci *gRPCCallInfoImpl) HTML() safehtml.HTML {
+	tmpl := template.Must(template.New("tcs").Parse("<b>Method:</b> {{.Method}}<b>Remote Addr:</b> {{.RemoteAddr}}"))
```

Comment:
move this to a package var that gets `Must`-ed at start time?

Reply:
Good one, fixed!